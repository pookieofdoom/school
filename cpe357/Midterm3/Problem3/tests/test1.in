checkSpace 0
newStack
checkSpace 0
isEmpty?
isStack? 0
done
