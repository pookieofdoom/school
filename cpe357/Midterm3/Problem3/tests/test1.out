checkSpace: Expected memory allocated
checkSpace: Expected memory allocated
isEmpty? TRUE
isStack? TRUE
